api/openapi-spec/swagger.json

```diff
@@ -16853,6 +16853,10 @@
       "description": "EvictionStrategy defines at the cluster level if the VirtualMachineInstance should be migrated instead of shut-off in case of a node drain. If the VirtualMachineInstance specific field is set it overrides the cluster level one.",
       "type": "string"
      },
+     "guestToRequestMemoryHeadroomRatio": {
+      "description": "GuestToRequestMemoryHeadroomRatio can be used to set a ratio between VM's memory guest and the memory allocated to the compute container. A higher ratio means that the VMs would be less compromised by node pressures, but would mean that fewer VMs could be scheduled to a node. If not set, the default is 1.",
```

Comment:
Please add a note if the internal overhead calculations are still taken into account or not (ignored)

Reply:
Will do.
FYI: they are.

Reply:
done